src/main/kotlin/com/google/androidstudiopoet/generators/DependencyGraphGenerator.kt

```diff
@@ -0,0 +1,166 @@
+package com.google.androidstudiopoet.generators
+
+import com.google.androidstudiopoet.models.*
+import com.google.androidstudiopoet.utils.joinPath
+import com.google.androidstudiopoet.writers.FileWriter
+import com.google.androidstudiopoet.writers.ImageWriter
+import java.awt.Color
+import java.awt.Graphics
+import java.awt.image.BufferedImage
+
+class DependencyGraphGenerator(private val fileWriter: FileWriter, private val imageWriter: ImageWriter) {
+
+    fun generate(projectBlueprint: ProjectBlueprint) {
+        val dependenciesGraphFileName = projectBlueprint.projectRoot.joinPath("dependencies.dot")
+        fileWriter.writeToFile(dependenciesGraphString(projectBlueprint), dependenciesGraphFileName)
+        println("Dependency graph written to $dependenciesGraphFileName")
+        DependencyImageGenerator(imageWriter).generate(projectBlueprint)
```

Comment:
Can you please inject `DependencyImageGenerator` instead of creating it here?

Reply:
What do you mean by inject? Pass as parameter?

Reply:
Done, also, moved DependencyImageGenerator to its own file, that movement simplified the tests by now having 2 test files also (that depend in a common base)